src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Inflector;
+
+/**
+ * Converts words between singular and plural forms.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ */
+final class Inflector
```

Comment:
@stof what do you think of this use case: https://github.com/symfony/symfony/pull/18260#discussion_r57116826 (just asking, I don't need that for now)